trpc/router/goal.ts

```diff
@@ -351,7 +377,9 @@ export const goal = router({
                 participants: true,
                 project: true,
                 tags: true,
-                estimate: true,
+                estimate: {
+                    include: { estimate: true },
```

Comment:
?

Reply:
from [this commit](https://github.com/taskany-inc/issues/commit/b531f6143e5c3c6de8342ac628fdcada3d26f3be) estimate relation has another one relation